effectiveOfficeBackend/src/main/kotlin/office/effective/feature/auth/service/UserService.kt

```diff
@@ -0,0 +1,33 @@
+package office.effective.feature.auth.service
+
+import office.effective.feature.auth.ITokenVerifier
+import office.effective.feature.auth.converters.UserDTOModelConverter
+import office.effective.feature.auth.dto.UserDTO
+import office.effective.feature.auth.repository.UserRepository
+import office.effective.model.UserModel
+import org.koin.core.context.GlobalContext
+
+class UserService() : IUserService {
+
+    private val verifier: ITokenVerifier = GlobalContext.get().get()
+    private val converterDTO: UserDTOModelConverter = GlobalContext.get().get()
+    private val repository: UserRepository = GlobalContext.get().get()
+
+    override fun getUsersByTag(tagStr: String): Set<UserDTO>? {
+        TODO("Not yet implemented")
+    }
+
+    override fun getUserById(userIdStr: String): UserDTO {
+        TODO("Not yet implemented")
+    }
+
+    override fun alterUser(user: UserDTO): UserDTO {
+        TODO("Not yet implemented")
+    }
```

Comment:
Write next to the *TODO* who they apply to (and a link to the board with the task if possible).